prow/cmd/deck/static/script.js

```diff
@@ -418,25 +422,50 @@ function redraw(fz) {
     redrawOptions(fz, opts);
 
     var lastKey = '';
-    for (var i = 0, emitted = 0; i < allBuilds.length && emitted < 500; i++) {
-        var build = allBuilds[i];
-        if (!equalSelected(typeSel, build.type)) {
+    for (var i = 0, emitted = 0; i < allBuilds.items.length && emitted < 500; i++) {
+        var build = allBuilds.items[i];
+
+        // assign default values 
+        var prop = { 
+            org: "",
+            repo: "",
+            base_ref: "", 
+            base_sha:"",
+            pulls: [
+                {
+                    "number": 0,
+                    "author": "",
+                    "sha": ""
+                }
+            ]
+        }
+ 
+        if ( Object.keys(build.spec.refs).length == 0){
```

Comment:
this should be explicitly `periodic` type job as we expect `postsubmit` `presubmit` and `batch` to have `refs`; otherwise, revision cells will be filled with broken links. 
 
If `postsubmit`, `presubmit` or `batch` does not have `refs`, should we consider to drop it?

Reply:
I am inclined to say just drop them but if we don't do that today I don't consider it important fixing here.